apps/prairielearn/src/pages/externalImageCapture/externalImageCapture.ts

```diff
@@ -52,8 +52,11 @@ export default function (options = { publicQuestionPreview: false }) {
   router.get(
     '/',
     asyncHandler(async (req, res) => {
-      if (!req.query.file_name) {
-        throw new HttpStatusError(400, 'file_name query parameter is required');
+      if (!req.query.file_name || !req.params.variant_id) {
```

Comment:
Why this change? I can't see when the variant ID would ever be missing, it's part of the URL. Ditto below.

Reply:
Done in ed32c0fd05e656102949ea224dc4336fbd3fba0f.